ConwaysGameOfLife.Tests/UniverseShould.cs

```diff
@@ -85,17 +87,17 @@ public void WhenInitialized_ReturnsEmptyListOfAliveCells()
         }
 
         [Fact]
-        public void testName() // test being used to test method 
+        public void testName() // TODO: rename
         {
             var initData = "XXX" + Environment.NewLine + 
                                     "XXX" + Environment.NewLine + 
                                     "XXX";

```

Comment:
Can we align the rows so that it's easier to see? I think the original suggestion on github messed up the alignment
![image](https://user-images.githubusercontent.com/13642138/120435510-2746fc00-c3c1-11eb-9f5e-7e3d84b99708.png)